test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Could you put this file in a subdirectory?

Reply:
done